projects/kalidea/kaligraphi/src/lib/atoms/kal-input/kal-input.component.html

```diff
@@ -12,6 +12,11 @@
     clear
   </kal-icon>
 
+  <kal-icon *ngIf="icon && !clearable"
+            (click)="customIconClicked()">
```

Comment:
iconClicked

Reply:
I can"t rename because it's the name of my event emitter